autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -225,7 +243,7 @@ class AutofillSettingsViewModel @Inject constructor(
 
     data class ViewState(
         val autofillEnabled: Boolean = true,
-        val logins: List<LoginCredentials> = emptyList(),
+        val logins: List<LoginCredentials>? = null,
```

Comment:
Because of the new "empty state" placeholder, it's important to be able to distinguish between "we have no saved credentials" and "we don't know yet how many there are".

Making this `nullable` supports the case whereby other properties of the view state have been initialised before we have the list of credentials.